methods_lab.js

```diff
@@ -0,0 +1,238 @@
+/*1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+```
+Input: [1,2,3,4,5]
+Output: [11,12,13,14,15]
+```*/
+
+
+function inBy10(arr){
+let newArr = arr.map((elem)=>{return elem + 10});
+return newArr}
+console.log(inBy10([1,2,3,4,5]));
+
```

Comment:
Did you know that you can return arr.map rather than defining it and then returning it?

function inBy10(arr){
	return arr.map((elem) => { 
                return elem + 10
      }
   );
}